torch/autograd/function.py

```diff
@@ -22,6 +23,10 @@
     "NestedIOFunction",
 ]
 
+# Unique id provider for each class inheriting from Function
+# This is incremented in FunctionMeta during class definition
+AUTOGRAD_FUNCTION_COUNTER = itertools.count()
```

Comment:
It is possible these could collide with the ID's used for AOTAutograd.

Reply:
added it separately as part of the key